src/views/SignUp.js

```diff
@@ -6,6 +6,7 @@ import { Modal, Button as AntButton, Form, Input, Typography } from 'antd';
 import { extractUserData, getUserFromRedirect, handleSignIn, saveUserInFirestore, updateUserName } from '../api';
 import styled from 'styled-components/macro';
 import queryString from 'query-string';
+import LoadingSpinner from '../components/LoadingSpinner';
```

Comment:
Can you add the component to the export list in `/components/LoadingSpinner.js`, and import it from there (line 4 in this file)?